### ISSUE TITLE
LoadApplicationClass Order

### ISSUE THREAD

**Issue by PhoenixFnX** [kenjis/ci-phpunit-test#71]
Hi,

I have an issue: I have a controller name which is used in a library I use. The CIPHPUNIT autoload feature loads libraries first to search for a class. I think Controller and models are more important than libraries which can be third party code.

I resolved it putting libraries scope below the others in this file CIPHPUnitTestAutoloader.php:71
```php
protected function loadApplicationClass($class)
	{
		$dirs = [
			APPPATH.'controllers',
			APPPATH.'models',
			APPPATH.'libraries',
		];
```

**Comment by kenjis** [comment]
In my opinion, models and libraries are more important than controllers. Libraries in CodeIgniter may be third party code, but may be *Domain Model*.

So I changed the order: https://github.com/kenjis/ci-phpunit-test/blob/master/application/tests/_ci_phpunit_test/CIPHPUnitTestAutoloader.php#L41

But now you can configure the search path:
See https://github.com/kenjis/ci-phpunit-test/blob/master/docs/HowToWriteTests.md#autoloader

**Comment by kenjis** [comment]
I released v0.9.0: https://github.com/kenjis/ci-phpunit-test/blob/master/docs/ChangeLog.md#v090-20151118
